CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## CHANGELOG
 
+### 0.1.6-SNAPSHOT
```

Comment:
I"m not sure how the release process works here. If I should have left this at 1.5 let me know.

Reply:
Actually I think I understand now, reverting this to 1.5-SNAPSHOT.